dsFunc.py

```diff
@@ -6,3 +6,28 @@ def elapsedTime(startTime, label):
     """Generate elapsed time."""
     td = datetime.timedelta(seconds=time.time() - startTime)
     print(label + ': {}'.format(td))
+    return td
+
+
+def instSelect():
+    """Select secrets.py file for the appropriate DSpace instance."""
+    sec = input('To edit production server, enter the name of the secrets '
+                'file: ')
+    if sec != '':
+        try:
+            secrets = __import__(sec)
+            print('Editing Production')
+        except ImportError:
+            secrets = __import__('secrets')
+            print('Editing Development')
+    else:
+        secrets = __import__('secrets')
+        print('Editing Development')
+
+    baseURL = secrets.baseURL
+    email = secrets.email
+    password = secrets.password
+    filePath = secrets.filePath
+    verify = secrets.verify
+    skipColl = secrets.skipColl
+    return(baseURL, email, password, filePath, verify, skipColl, sec)
```

Comment:
This is quite a lot of return values. I'd worry about messing up the order when I called it. If you end up with the correct cardinality of return values, but in a different order you may end up with subtle bugs that aren't immediately obvious. Returning a tuple also tightly couples the calling of this function with its internals. If you decide to change the order of these, or add or remove one, you'll have to update every instance of this function call. The easiest solution in this case would probably be to just return the `secrets` variable itself.

Reply:
Ah, a very good point. I didn't realize the order made a difference because I haven't used tuples that much. I will reconsider this and update the PR. Thank you!